File: modules/smarthome/http/dummyurl1.py
#!/usr/bin/python3
import sys
import os
import time
numbers=sys.argv[1]
number=int(numbers)
basePath = '/var/www/html/openWB'
try:
    f = open(basePath+'/ramdisk/device' + str(number) + '_req_relais', 'r')
    status =int(f.read())
    f.close()
except Exception as e:
    status = 0
status = status * 100
print(status)

File: packages/modules/smarthome/http/dummyurl1.php
<?php
$dec = $_REQUEST['d'];
$file = "/var/www/html/openWB/ramdisk/device" . $dec . "_req_relais";
if (is_file($file)) {
	$a = file_get_contents($file);
	if ($a == "1") {
		echo 100;
	} else { echo 0; }
} else { echo 0; }
?>
